kani-compiler/src/kani_middle/transform/check_uninit/ptr_uninit/mod.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! A transformation pass that instruments the code to detect possible UB due to the accesses to
```

Comment:
Is this code new or is it old code that you moved to its own module?

Reply:
This is old code moved into its own module